src/halfpipe/ui/setting/vals.py

```diff
@@ -49,6 +49,8 @@ class ConfoundsSelectStep(Step):
             "Derivatives of motion parameters": "(trans|rot)_[xyz]_derivative1",
             "Motion parameters squared": "(trans|rot)_[xyz]_power2",
             "Derivatives of motion parameters squared": "(trans|rot)_[xyz]_derivative1_power2",
+            "Frame-wise displacement threshold": "fd-spike-threshold",
+            "DVARS Spike Threshold": "dvars-spike-threshold",
```

Comment:
```suggestion
            "Motion scrubbing": "motion_outlier[0-9]+",
```
I think we might have mixed up two different things in our previous meeting. The motion outlier columns are calculated based on the threshold, but are then just named `motion_outlier00`, `motion_outlier01`, etc. in the confounds file. I think that means we need to just use a single regular expression for those